calico_kubernetes/calico_kubernetes.py

```diff
@@ -56,20 +65,19 @@ def __init__(self):
         self.profile_name = None
         self.namespace = None
         self.docker_id = None
+        self.policy_parser = None
 
         self._datastore_client = IPAMClient()
-        self.calicoctl = sh.Command(CALICOCTL_PATH).bake(_env=os.environ)
         self._docker_client = Client(
             version=DOCKER_VERSION,
             base_url=os.getenv("DOCKER_HOST", "unix://var/run/docker.sock"))
 
     def create(self, namespace, pod_name, docker_id):
         """"Create a pod."""
-        # Calicoctl does not support the '-' character in iptables rule names.
-        # TODO: fix Felix to support '-' characters.
         self.pod_name = pod_name
         self.docker_id = docker_id
         self.namespace = namespace
```

Comment:
Can we rename this `k8s_namespace` or something like that? `namespace` is overloaded with `network_namespace`.